in_text/text_04_inputs.yml

```diff
@@ -378,172 +381,174 @@ entities:
       data-max: 2020-07-31
       data-units: NA
     -
+      # 
       attr-label: seg_ccov
       attr-def: >-
         Area-weighted average cloud cover fraction for each segment from HRUs contributing flow to the segment.
       attr-defs: https://doi.org/10.1002/joc.3413
-      data-min: 0.2565
-      data-max: 0.8
+      data-min: 0.2542
+      data-max: 7667
       data-units: decimal fraction
     -
+      #
       attr-label: seg_humid
       attr-def: >-
         Area-weighted average relative humidity for each segment from HRUs contributing flow to the segment
       attr-defs: https://doi.org/10.1002/joc.3413
-      data-min: 0.7
-      data-max: 0.7
+      data-min: 0
+      data-max: 0
       data-units: decimal fraction
     -
       attr-label: seg_outflow
       attr-def: >-
         Streamflow leaving a segment
       attr-defs: This data release
-      data-min: 1.50589e-05
-      data-max: 838.4617
+      data-min: 0
+      data-max: 3086.53
       data-units: cubic meters per second
     -
       attr-label: seg_rain
       attr-def: >-
         Area-weighted average rainfall for each segment from HRUs contributing flow to the segment
       attr-defs: https://doi.org/10.1002/joc.3413
       data-min: 0.0
-      data-max: 0.160782
+      data-max: 1587.5
```

Comment:
Thanks for looking into it. Maybe we can re-store the model output but don't include `3558` in the output because it appears to be giving bad values for several variables